### PR TITLE
Fix typos in Theming doc

### PR DESCRIPTION
**Pull request by DjebbZ**
- typo
- no error page in example, so I removed the mention in the text
- I added a comment in the text itself, another occurrence of something mentioned in text that doesn't appear in code.

So I guess you have some fixing to do before merging.

### CODE REVIEW

File: docs/Theming.md [hunk 1]
```diff
@@ -890,7 +890,7 @@ const App = () => (
 
 The `MenuItemLink` component make use of the React Router [NavLink](https://reacttraining.com/react-router/web/api/NavLink) component, hence allowing to customize its style when it targets the current page.
 
-If the default active style does not suit your tastes, you can override it by passing your own `classes`:
+If the default active style does not suit your tastes, you can override it by passing your own `classes`: // the code below doesn't use the `classes` prop
```
Comment (fzaninotto) [inline docs/Theming.md:893]: The code below, introduced in 8fcd609e53ca1ff9ec1cf3635b7cdf2b5b42f5f3, doesn't teach anything about classes and doesn't show anything more than the previous snippet. It and should be removed, together with the entire sentence "if the default active style...".

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Theming.md [hunk 2]
```diff
@@ -387,7 +387,7 @@ const App = () => (
 );
 ```
 
-Your custom layout can extend the default `<Layout>` component if you only want to override the sidebar, the appBar, the menu, the notification component, or the error page. For instance:
+Your custom layout can extend the default `<Layout>` component if you only want to override the sidebar, the appBar, the menu or the notification component. For instance:
```
[the remark this answers is not in the document]
Comment (DjebbZ) [reply docs/Theming.md]: Done in https://github.com/marmelab/react-admin/pull/5546/commits/63c2d5f25a4622bae52f52b5d5a6d6d17e042120

File: docs/Theming.md (hunk 1 above)
Comment (DjebbZ) [reply docs/Theming.md]: Done in https://github.com/marmelab/react-admin/pull/5546/commits/07e7ed26eeec8de6155988e04f43247d5b3d6feb

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**